src/app/components/filter/filter-panel.jsx

```diff
@@ -23,21 +23,34 @@ const useStyles = makeStyles({
     background: '#fafdff',
     display: 'grid',
     gridTemplateColumns: '1fr',
-    padding: '16px',
     rowGap: '16px',
+    maxHeight: '64vh',
     [theme.breakpoints.up('sm')]: {
       gridTemplateColumns: '1fr 1fr',
     },
   },
+  filters:{
+    height: '100%',
+    borderRight: '1px solid #599bde',
+    padding: '16px',
+    zIndex: 15,
+  },
+  subfilters:{
```

Comment:
I'd like to standardize on some terminology with these. The primary panel has an ID of filter-panel-primary, and the secondary one is filter-panel-secondary. I don't know if those IDs will work as CSS selectors, but the primary/secondary should work for class names, in some fashion